src/System/Wlog/Exception.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE ExplicitForAll      #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- | This module deals with Exception logging.
+
+module System.Wlog.Exception
+       ( logException
+       , catchLog
+       ) where
+
+import Universum
+
+import Control.Exception.Base (Exception)
+
+import System.Wlog.CanLog (WithLogger, WithLoggerIO, logError)
+
+-- | Logs exception's description with ''System.Wlog.Severity.Error' 'System.Wlog.Severity.Severity'
+logException :: (WithLogger m, Exception e) => e -> m ()
```

Comment:
I think, you need to specify `e` as a first type variable with `forall`. I expect that in future users might want to use `-XTypeApplications` to specify exception to log.

Reply:
Yes, true! This definitely should be done in right order..